kontonummer.js

```diff
@@ -425,4 +484,27 @@
         }
         return sum && sum % 11 === 0;
     };
-}(window));
+
+    /**
+     * Fyll nollor för Swedbank bankkontonumren
+     *
+     * @param {String} number Bankkontonummer
+     * @param {Object} bank Bank object
+     * @returns {String} Bankkontonummer med nollor
+     */
+    var fillZeros = function (accountNumber, bank) {
+        var numberWithoutClearing = accountNumber.substr(bank.lengths.clearing);
+        while (numberWithoutClearing.length < bank.lengths.account) {
+            numberWithoutClearing = '0' + numberWithoutClearing;
```

Comment:
While I realize `Array.repeat`, `String.padStart` and `String.repeat()` are not supported in IE, I think the `Array(accountNumberLength + 1).join('0')` would be a well supported cross-browser alternative to a `while`-loop, so in context:
```js
var clearingNumber = accountNumber.substr(0, bank.lengths.clearing;
var numberWithoutClearing = accountNumber.substr(bank.lengths.clearing);
var accountNumberLength = bank.lengths.account - numberWithoutClearing.length;
var zeroFill = Array(accountNumberLength + 1).join('0');
return clearingNumber.concat(zeroFill, numberWithoutClearing);
```